build.cake

```diff
@@ -116,87 +116,87 @@ Task("Build")
-			MSBuild(GUI_SOLUTION, new MSBuildSettings()
-				.SetConfiguration(configuration)
-				.SetVerbosity(Verbosity.Minimal)
-				.SetNodeReuse(false)
-				.SetPlatformTarget(PlatformTarget.MSIL)
-			);
-		}
-		else
-		{
-			// Use XBuild
-			XBuild(GUI_SOLUTION, new XBuildSettings()
-				.WithTarget("Build")
-				.WithProperty("Configuration", configuration)
-				.SetVerbosity(Verbosity.Minimal)
-			);
-		}
+        if(IsRunningOnWindows())
+        {
+            // Use MSBuild
+            MSBuild(GUI_SOLUTION, new MSBuildSettings()
+                .SetConfiguration(configuration)
+                .SetVerbosity(Verbosity.Minimal)
+                .SetNodeReuse(false)
+                .SetPlatformTarget(PlatformTarget.MSIL)
+            );
+        }
+        else
+        {
+            // Use XBuild
+            XBuild(GUI_SOLUTION, new XBuildSettings()
+                .WithTarget("Build")
+                .WithProperty("Configuration", configuration)
+                .SetVerbosity(Verbosity.Minimal)
+            );
+        }
     });
 
 //////////////////////////////////////////////////////////////////////
 // TEST
 //////////////////////////////////////////////////////////////////////
 
 Task("Test")
-	.IsDependentOn("Build")
-	.Does(() =>
-	{
-		NUnit3(GUI_TESTS);
-	});
+    .IsDependentOn("Build")
+    .Does(() =>
+    {
+        NUnit3(GUI_TESTS);
+    });
 
 //////////////////////////////////////////////////////////////////////
 // PACKAGE
 //////////////////////////////////////////////////////////////////////
 
 Task("PackageZip")
-	.IsDependentOn("Build")
-	.Does(() =>
-	{
-		CreateDirectory(PACKAGE_DIR);
-
-		CopyFileToDirectory("LICENSE", BIN_DIR);
-		CopyFileToDirectory("CHANGES.txt", BIN_DIR);
-
-		var zipFiles = new FilePath[]
-		{
-			BIN_DIR + "LICENSE",
-			BIN_DIR + "CHANGES.txt",
-			BIN_DIR + "nunit-gui.exe",
-			BIN_DIR + "nunit-gui.exe.config",
-			BIN_DIR + "nunit-gui.pdb",
-			BIN_DIR + "nunit.uikit.dll",
-			BIN_DIR + "nunit.uikit.pdb",
-			BIN_DIR + "nunit.engine.api.dll",
-			BIN_DIR + "nunit.engine.dll",
-			BIN_DIR + "nunit.engine.addins",
-			BIN_DIR + "Mono.Cecil.dll"
-		};
-
-		Zip(BIN_DIR, File(ZIP_PACKAGE), zipFiles);
-	});
+    .IsDependentOn("Build")
+    .Does(() =>
+    {
+        CreateDirectory(PACKAGE_DIR);
+
+        CopyFileToDirectory("LICENSE", BIN_DIR);
+        CopyFileToDirectory("CHANGES.txt", BIN_DIR);
+
+        var zipFiles = new FilePath[]
+        {
+            BIN_DIR + "LICENSE",
+            BIN_DIR + "CHANGES.txt",
+            BIN_DIR + "nunit-gui.exe",
+            BIN_DIR + "nunit-gui.exe.config",
+            BIN_DIR + "nunit-gui.pdb",
+            BIN_DIR + "nunit.uikit.dll",
+            BIN_DIR + "nunit.uikit.pdb",
+            BIN_DIR + "nunit.engine.api.dll",
+            BIN_DIR + "nunit.engine.dll",
+            BIN_DIR + "nunit.engine.addins",
+            BIN_DIR + "Mono.Cecil.dll"
+        };
+
+        Zip(BIN_DIR, File(ZIP_PACKAGE), zipFiles);
+    });
```

Comment:
In general, don't mix changes like this in with substantive changes to other files, especially when working cooperatively. We can stand this one, but see other comments.